ibm_cloud_sdk_core/authenticators/iam_request_based_authenticator.py

```diff
@@ -15,11 +15,13 @@
 # limitations under the License.
 
 from typing import Dict
-
```

Comment:
This empty line should be there to separate std lib and 3rd party package imports. :)

Reply:
fixed